windows/src/desktop/setup/Keyman.Setup.System.InstallInfo.pas

```diff
@@ -390,6 +391,32 @@ procedure TInstallInfo.LocatePackagesAndTierFromFilename(Filename: string);
   else if p.Equals(SKeymanSetup_Beta) then FTier := TIER_BETA
   else if p.Equals(SKeymanSetup_Alpha) then FTier := TIER_ALPHA;
 
+  // There is at least one browser that appends `_n` to a filename for
+  // duplicate downloads. This is a little tricky to handle because `_n`
```

Comment:
Out of curiosity, what browser does this?

Reply:
I haven't discovered which browser is doing this yet. Just observing a reasonably common pattern in the crash reports.